include/deal.II/numerics/vector_tools.templates.h

```diff
@@ -234,74 +236,124 @@ namespace VectorTools
         function.vector_value_list(generalized_support_points,
                                    function_values);
 
-        // FIXME: In case of an FESystem we have to apply this
-        // transformation according to the conformity of each base element.
+        // A small helper function to transform a component range starting
+        // at offset from the real to the unit cell according to the
+        // supplied conformity.
```

Comment:
maybe say "a component range starting at 'offset' and with 'dim' components"

Reply:
Also, can you say *which variable you are operating one*? Would it be worth only capturing that variable, or maybe passing it as a variable?